pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -540,9 +540,37 @@ var (
 					Schema: &EndpointSelector,
 				},
 			},
+			"toGroups": {
+				Description: `ToGroups is a list of constraints that will
+				gather data from third-party providers and create a new
+				derived policy.`,
+				Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
+					"AWS": AWSGroup,
+				},
+			},
 		},
 	}
 
+	AWSGroup = apiextensionsv1beta1.JSONSchemaProps{
+		Description: "",
+		Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
+			"SecurityGroupsIds": {
+				Description: `SecurityGroupsIds is the list of AWS security
+				group IDs that will filter the instances IPs from the AWS API`,
+				Type: "array",
```

Comment:
Can be done as a follow up but I would like to see the validation done for the `groupIDs` object. There must be a defined regex in aws docs for this.